src/Symfony/Component/Routing/RequestContext.php

```diff
@@ -23,18 +23,19 @@
  */
 class RequestContext
 {
-    private string $baseUrl;
     private string $pathInfo;
-    private string $method;
-    private string $host;
-    private string $scheme;
-    private int $httpPort;
-    private int $httpsPort;
-    private string $queryString;
     private array $parameters = [];
 
-    public function __construct(string $baseUrl = '', string $method = 'GET', string $host = 'localhost', string $scheme = 'http', int $httpPort = 80, int $httpsPort = 443, string $path = '/', string $queryString = '')
-    {
+    public function __construct(
+        private string $baseUrl = '',
+        private string $method = 'GET',
+        private string $host = 'localhost',
+        private string $scheme = 'http',
+        private int $httpPort = 80,
+        private int $httpsPort = 443,
+        string $path = '/',
+        private string $queryString = '',
```

Comment:
Using promoted properties does not make sense here as the constructor was not assigning the properties directly (and so the new code is not equivalent)